yarn.lock

```diff
@@ -8217,6 +8222,15 @@ json-stringify-safe@~5.0.1:
   resolved "https://registry.yarnpkg.com/json-stringify-safe/-/json-stringify-safe-5.0.1.tgz#1296a2d58fd45f19a0f6ce01d65701e2c735b6eb"
   integrity sha1-Epai1Y/UXxmg9s4B1lcB4sc1tus=
 
+json2csv@^5.0.7:
```

Comment:
This will fall off when this PR https://github.com/growthbook/growthbook/pull/396 goes live.